application/src/main/java/org/thingsboard/server/service/ws/notification/DefaultNotificationCommandsHandler.java

```diff
@@ -242,7 +242,7 @@ public void handleMarkAllAsReadCmd(WebSocketSessionRef sessionRef, MarkAllNotifi
 
     @Override
     public void handleUnsubCmd(WebSocketSessionRef sessionRef, UnsubscribeCmd cmd) {
-        localSubscriptionService.cancelSubscription(sessionRef.getSessionId(), cmd.getCmdId());
+        localSubscriptionService.cancelSubscription(sessionRef.getSecurityCtx().getTenantId(), sessionRef.getSessionId(), cmd.getCmdId());
```

Comment:
Let's introduce a method `sessionRef.getTenantId()` and replace usages of `sessionRef.getSecurityCtx().getTenantId()` with it.
Also, please check in that method that securityCtx is not null (it will be null until user sends auth cmd), otherwise return sys tenant id.